docs/docker_based_installation.rst

```diff
@@ -1,24 +1,23 @@
 Installation based on Docker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Since 1.7.4 mss is on the `docker hub <https://hub.docker.com/orgs/openmss/>`_.
+You can use images from the `docker hub <https://hub.docker.com/orgs/openmss/>`_. based on our `repository <https://github.com/Open-MSS/dockerhub>`_
+
+Build settings are based on the stable branch. Our openmss/mss:latest has any update in the stable branch.
 
-Build settings are based on the stable branch. Our latest is any update in the stable repo.
 
 You can start server and client by loading the image ::
 
  $ xhost +local:docker
- $ docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix/:/tmp/.X11-unix dreimark/mss:latest  /bin/bash
+ $ docker run -ti --rm -e DISPLAY=$DISPLAY -v /tmp/.X11-unix/:/tmp/.X11-unix openmss/mss:latest  /bin/bash
  $ /opt/conda/envs/mssenv/bin/mss &
  $ /opt/conda/envs/mssenv/bin/mswms --port 80 &
  $ /opt/conda/envs/mssenv/bin/mscolab &
  $ curl http://localhost/?service=WMS&request=GetCapabilities&version=1.1.1
- $ curl http://localhost:8083
+ $ curl http://localhost:8083/status
 
-The userinterface and both servers can be started by ::
+The WMS server initialized by demodata, and the mscolab server and the userinterface can be started by ::
```

Comment:
> The WMS server initialized by demodata, and the mscolab server and the userinterface

I think removing the first "and"  would make this sentence nicer. Though I suppose it really isn't important.

Reply:
ok, later